TELEMETRY.md

```diff
@@ -1,48 +1,3 @@
-# HttpRepl telemetry
+# HTTP REPL telemetry
```

Comment:
We need to get a ruling on this from someone (@bradygaster ?) - the tool itself mostly refers to itself as HttpRepl, but the docs mostly say HTTP REPL. For this one change, it doesn't matter. But we should make a pass for consistency once we find a way to standardize it.

Reply:
HttpRepl is my vote